setup.py

```diff
@@ -54,6 +54,7 @@
         'Operating System :: OS Independent',
         'Programming Language :: JavaScript',
         'Programming Language :: Python',
+        'Framework :: Django',
```

Comment:
We'll want specific versions here, but we'll be dropping a few shortly so that can wait.